lms/djangoapps/teams/serializers.py

```diff
@@ -171,6 +171,13 @@ class BaseTopicSerializer(serializers.Serializer):
     name = serializers.CharField()
     id = serializers.CharField()  # pylint: disable=invalid-name
 
+    # Required overrides of abstract base class methods, but we don't use them
+    def create(self, validated_data):
+        pass
+
+    def update(self, instance, validated_data):
+        pass
+
```

Comment:
Cases like this where we're just adding unused code in order to make pylint happy are where it's better to just disable the warning for that class. You can disable the warning for just the class if you put the pylint-disable within the body of the class.